packages/site/src/pages/index.tsx

```diff
@@ -417,6 +433,25 @@ const Index = () => {
       },
       successMessage: 'Smart Contract Account Created',
     },
+    {
+      name: 'Create account (Deterministic)',
+      description:
+        'Create a 4337 account using a deterministic key generated through the snap',
+      inputs: [
+        {
+          id: 'create-account-deterministic',
+          title: 'Counter',
+          value: counter.toString(),
+          type: InputType.TextField,
+          onChange: () => {},
+        },
```

Comment:
Looks good but why do we need to show the input field?

Reply:
let's have single button called create new account!

Reply:
Since this is a demo page, and there are two account generation methods - displaying the counter is to help users when they are regenerating older deterministic accounts(and felt the easiest way for me to show the difference). This ofc is just for this demo page, on the actual page we'll have to show up account cards/addresses (already added with each type) and then a button to add new account.